resources/jaeger/README.md

```diff
@@ -0,0 +1,16 @@
+# Jaeger
+
+## Overview
+[Jaeger](http://jaeger.readthedocs.io/en/latest/) is a monitoring and tracing tool for microservices-based distributed systems.
+
+## Details
+The Envoy sidecar uses Jaeger to trace the request flow in the Istio service mesh. The communication from Istio and Envoy uses the Zipkin protocol. Jaeger provides compatibility with the Zipkin protocol. This allows you to use Zipkin protocol and clients in Istio, Envoy, and the Kyma services.
```

Comment:
Can we have here that:
`Istio and Envoy use the Zipkin protocol to communicate with the tracing back-end.` ?

Reply:
Done.